app/src/main/java/me/guillaumin/android/osmtracker/db/model/TrackStatistics.java

```diff
@@ -0,0 +1,246 @@
+package me.guillaumin.android.osmtracker.db.model;
+
+import me.guillaumin.android.osmtracker.db.TrackContentProvider;
+import me.guillaumin.android.osmtracker.db.TrackContentProvider.Schema;
+import android.content.ContentResolver;
+import android.database.Cursor;
+import android.location.Location;
+import android.util.Log;
+
+/**
+ * Represents statistics, such as  total length and maximum speed, for a Track
+ *
+ * @author Arseniy Lartsev
+ *
+ */
+public class TrackStatistics {
+	private long trackId;
+	private long pointCount;
+	private float length;
+	private float maxSpeed;
+	private double lastLatitude;
+	private double lastLongitude;
+	private long lastTime;
+	private float timeMoving;
+
+	/**
+	 * Copypasta from osmand
```

Comment:
Yeah apparently not, that's disappointing. In that case I think the best practice is to take the [whole class file](https://github.com/osmandapp/Osmand/blob/6065d44c9a0eb85fe450e5da245d4965db16951e/OsmAnd-java/src/net/osmand/Location.java) and put it under the same package to make it clear that it's coming from a separate project.